unittests/tools/test_harbor_vulnerability_parser.py

```diff
@@ -44,3 +44,13 @@ def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
         self.assertEqual(finding.severity, 'High')
         self.assertIsNone(finding.mitigation)
         self.assertIsNone(finding.references)
+
+    # Sample with Trivy Test
+    def test_parse_file_with_multiple_vuln_has_multiple_trivy_findings(self):
+        testfile = open("unittests/scans/harbor_vulnerability/harbor-trivy-vuln.json")
+        parser = HarborVulnerabilityParser()
+        findings = parser.get_findings(testfile, Test())
+
+        finding = findings[0]
+        self.assertEqual(finding.severity, 'High')
+        self.assertEqual(finding.cwe, '125')
```

Comment:
Why do I need a string here for finding.cwe and in the veracode an integer:
https://github.com/DefectDojo/django-DefectDojo/blob/9742d9ff21b36ef7daf6b927ed5ac822c8227c49/unittests/tools/test_veracode_parser.py#L30C18-L30C18 

`self.assertEqual(123, finding.cwe)`

Reply:
Also here there is an example equally to veracode:
https://defectdojo.github.io/django-DefectDojo/contributing/how-to-write-a-parser/#tests-api-parsers